.github/workflows/check_docker.yml

```
@@ -2,7 +2,7 @@ name: Check Docker
 
 on:
   push:
-    branches: [main, holoscan-sdk-lws2]
+    branches: [main, holoscan-sdk-lws2, update-to-use-cli]
```

remove this before merge

```suggestion
    branches: [main, holoscan-sdk-lws2]
```